x/crosschain/client/cli/cli_cctx.go

```diff
@@ -277,3 +277,25 @@ func CmdCCTXOutboundVoter() *cobra.Command {
 
 	return cmd
 }
+
+func CmdAbortStuckCCTX() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "abort-stuck-cctx [index]",
+		Short: "abort a stuck CCTX",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx := client.GetClientContextFromCmd(cmd)
+
+			msg := &types.MsgAbortStuckCCTX{
+				Creator:   clientCtx.GetFromAddress().String(),
+				CctxIndex: args[0],
+			}
+
```

Comment:
maybe add validatebasic here

Reply:
It get called in `GenerateOrBroadcastTxCLI` actually